google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -129,19 +129,19 @@ public void actionPerformed(ActionEvent e) {
       }
     });
 
-    myWarningLabel.setVisible(false);
-    myWarningLabel.setFont(new Font(myWarningLabel.getFont().getName(), Font.BOLD,
-                                    myWarningLabel.getFont().getSize() - 1));
-    myWarningLabel.setForeground(JBColor.RED);
-    myWarningLabel2.setVisible(false);
-    myWarningLabel2.setFont(new Font(myWarningLabel2.getFont().getName(), Font.PLAIN,
-                                     myWarningLabel.getFont().getSize() - 1));
-    myWarningLabel2.setText(GctBundle.getString("clouddebug.sourcedoesnotmatch"));
```

Comment:
to be more concise, it can have things like "..source does not match..", "..Could not locate a local / remote Git repository.." and so on. I guess we could call it something like "repositoryWarningMessage" or something like that, but that's a bit of a stretch.